test/action_image_client.cpp

```diff
@@ -0,0 +1,66 @@
+#include <actionlib/client/simple_action_client.h>
+#include <camera_control_msgs/GrabImagesAction.h>
+#include <image_transport/image_transport.h>
+#include <ros/ros.h>
+
+typedef actionlib::SimpleActionClient<camera_control_msgs::GrabImagesAction>
+    ActionImageActionClient;
+typedef boost::shared_ptr<ActionImageActionClient> ActionImageActionClientPtr;
```

Comment:
cool that you use smart pointers, but you don't share the action server. please go for a unique_ptr so the ownership is easily trackable